packages/boiler-task-selenium/src/runner/webdriverio/index.js

```diff
@@ -9,4 +9,11 @@ require('babel-register')({
 });
 require('babel-polyfill');
 
-exports.config = require('./config.js');
+const opts = {};
+if (process.argv[4] === '--tunnel') {
+  Object.assign(opts, {
+    maxInstances: 5,
+    reporters: '[dot]'
+  });
+}
+exports.config = require('./make-config.js').default(opts);
```

Comment:
is this how we're detecting parallel and switching the reporters? Seems like we should be checking for a `TRAVIS_BRANCH` or looking at the the `opts.length > 1` or `opts[0].capabilities.length`. On top of this, if we passed some config into the `spawn` function that tells it if this is a `tunnel`, `remote` or `local` we can pass that as an ENV variable to the child process. Otherwise, we will be potentially will be running the spec reported in CI.